src/composables/fungibleTokens.ts

```diff
@@ -290,6 +290,9 @@ export function useFungibleTokens() {
       if (val.length !== oldVal.length) {
         loadTokenBalances();
       }
+      if (val.filter(({ protocol }) => protocol === val.at(-1)?.protocol).length === 1) {
```

Comment:
1. I suggest to add small comment that this is to check if the new account is the first for specified protocol.
2. Also I guess this test might be put inside of the previous test where we test if there is a new account added.